src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,16 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+
 public class UserService {
 
     public int getUserScore(String[] records, String email) {
-        return 0;
+        for (String str : records) {
+            String[] temp = str.split(":"); //Подскажите бест-практис, пожалуйста
+            if (temp[0].equals(email)) {
```

Comment:
ideally, you may make 0 and 1 constants so it will be obvious what information is located under each index